core/src/test/java/org/hibernate/ogm/backendtck/optimisticlocking/OptimisticLockingExtraTest.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.backendtck.optimisticlocking;
+
+import java.io.Serializable;
+
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.ogm.cfg.OgmProperties;
+import org.hibernate.ogm.datastore.spi.DatastoreProvider;
+import org.hibernate.ogm.dialect.impl.ForwardingGridDialect;
+import org.hibernate.ogm.dialect.spi.TupleContext;
+import org.hibernate.ogm.model.key.spi.EntityKey;
+import org.hibernate.ogm.model.spi.Tuple;
+import org.hibernate.ogm.utils.GridDialectType;
+import org.hibernate.ogm.utils.OgmTestCase;
+import org.hibernate.ogm.utils.SkipByGridDialect;
+import org.hibernate.ogm.utils.TestHelper;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+
+/**
+ * Test for detecting concurrent updates by dialects which support atomic find/update semantics or have their own
+ * optimistic locking scheme.
+ *
+ * @author Gunnar Morling
+ */
+@SkipByGridDialect(
+		value = { GridDialectType.CASSANDRA },
+		comment = "Galaxy.stars list - bag semantics unsupported (no primary key)"
```

Comment:
Can we avoid to add the name of the class in the comment? It's very easy to forget about it if we rename forget about something.

It think, saying bag semantic not supported or collection of embeddable not supported is enough since our entity classes are aimed to test a specific scenario.